### PR TITLE
Adding a new PHP array

### PR DESCRIPTION
**Pull request by JayRKyd**
### Description

The array_values() function takes in an array and returns a new array in numerical order. It effectively removes keys from the array and re-indexes it, starting from zero. The reason for creating this doc is to help beginners understand PHP functions properly.

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-values/array-values.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+Tags:
+  - 'Arrays'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_values()`** function allows you to return all the values from an array in numerical order.
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-values/array-values.md:16]:
```suggestion
The **`array_values()`** function returns all the values from an array in numerical order.
```

File: content/php/concepts/arrays/terms/array-values/array-values.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+```
+array_values(array)
+```
+
+> **Note:** The array that is returned will have numeric keys that start from 0 and is incremented by 1.
+
+## Example
+The example shown below creates an array called `$vehicle` with three key-value pairs. It then uses the `array_values()` function to extract and print the array's values resulting in an indexed array.
+```
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-values/array-values.md:28]:
````suggestion
## Example

The example shown below creates an array called `$vehicle` with three key-value pairs. It then uses the `array_values()` function to extract and print the array's values resulting in an indexed array.

```php
````

File: content/php/concepts/arrays/terms/array-values/array-values.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_values()`** function allows you to return all the values from an array in numerical order.
+
+## Syntax
+
+```
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-values/array-values.md:20]:
````suggestion
```pseudo
````

File: content/php/concepts/arrays/terms/array-values/array-values.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+## Example
+The example shown below creates an array called `$vehicle` with three key-value pairs. It then uses the `array_values()` function to extract and print the array's values resulting in an indexed array.
+```
+<?php
+$vehicle = array("car" => "BMW", "color" => "gold", "model" => "X5");
+print_r(array_values($vehicle));
+?>
+```
+This show the values "BMW", "gold" and "X5" that are printed to the console.
+```shell
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-values/array-values.md:35]:
````suggestion
```

This example shows the values "BMW", "gold" and "X5" printed to the console:

```shell
````

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@JayRKyd I did a first review and left a few comments.

**Comment by caupolicandiaz** [comment]
@JayRKyd It has been several weeks with no activity on this entry. Are you still available to edit this?

**Review by caupolicandiaz — COMMENTED** [review]
@JayRKyd @SSwiniarski For the sake of moving this along I've gone ahead and incorporated all of the edits.

**Review by SSwiniarski — APPROVED** [review]
@JayRKyd, @caupolicandiaz Looks good.

**Review by caupolicandiaz — APPROVED** [review]
@JayRKyd @SSwiniarski This is good to merge

**Pull request merged**